docs/pages/getting-started/digitalocean.mdx

```diff
@@ -4,52 +4,51 @@ description: How to install Teleport on DigitalOcean?
 videoBanner: voHQlSX_czE
 ---
 
-This tutorial will guide you through quickly getting started with Teleport on DigitalOcean with Teleport 1-Click Droplet app.
- 
+
 <Admonition
   type="note"
   title="Note"
 >
-  This tutorial covers getting started with Teleport using DigitalOcean 1-Click droplet app. If you are looking for a manual installation, refer to our [Linux installation guide](./linux-server.mdx). 
+This tutorial will guide you through quickly getting started with Teleport on DigitalOcean with Teleport 1-Click Droplet app. If you are looking for a manual installation, refer to our [Linux installation guide](./linux-server.mdx). 
 
 </Admonition>
 
-Prerequisites
-- A Fully Qualified Domain Name (this will be used as a [clustername](../setup/admin/trustedclusters.mdx#introduction) for Teleport).
-- A two-factor authenticator app(e.g., [Google Authenticator](https://www.google.com/landing/2step/)).
+**Prerequisites**
+- A Fully Qualified Domain Name (FQDN).
+- A two-factor authenticator app (e.g., [Google Authenticator](https://www.google.com/landing/2step/)).
 - An SSH client like OpenSSH.
 
-## Step 1/4. Install Teleport with 1-Click
+## Step 1/3. Install Teleport with 1-Click
 
-Head over to Teleport page on [DigitalOcean marketplace](https://marketplace.digitalocean.com/apps/teleport) and click “Create a Droplet” button:
+Head over to the Teleport page on [DigitalOcean marketplace](https://marketplace.digitalocean.com/apps/teleport) and click the “Create a Droplet” button:
```

Comment:
```suggestion
Head over to the Teleport page on [DigitalOcean Marketplace](https://marketplace.digitalocean.com/apps/teleport) and click the “Create a Droplet” button:
```

I think this is capitalized (https://www.digitalocean.com/blog/introducing-digitalocean-marketplace/).